tests/unit_tests/commandline/test_log_setup.py

```diff
@@ -84,9 +84,9 @@ def generate_random_messages(add_long_word=False):
     if add_long_word:
 
         def f(s):
-            insert_random_string_randomly(s, 88)  # insert string 88 chars long
+            return insert_random_string_randomly(s, 88)  # insert string 88 chars long
```

Comment:
Oh gosh! Can't believe this got through review. Thanks for fixing

Reply:
No problem!